packages/default-theme/cms/CmsGenericBlock.vue

```diff
@@ -1,34 +1,34 @@
 <template>
-  <component :is="getComponent" :content="content" :style="slotStyles"/>
+  <component :is="getComponent" :content="content" :style="slotStyles" />
 </template>
 
 <script>
-import { getComponentBy } from "../cmsNameMapper";
+import { getCmsBlockComponent } from 'sw-cms/cmsNameMapper'
```

Comment:
`...from @shopware-pwa/default-theme/sw-cms/cmsNameMapper` maybe?

Reply:
@mkucmus I think it gets resolved anyway via nuxt

Reply:
'sw-cms/cmsNameMapper' file is dynamically created in `.shopware-pwa/sw-cms/cmsNameMapper` path. Thanks to this we can easily resolve CMSes.
Folder `sw-cms` does not exist in default-theme nor will exist in any theme